### PR TITLE
Create DAO UI and its depoyApp feature

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
image

### CODE REVIEW

File: src/contract/dao/dao-app-clear.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { compileToUnit8Array } from "@/utility";
+
+const daoAppClear = ` #pragma version 6
```
Comment (rmeena840) [inline src/contract/dao/dao-app-clear.ts]:
```suggestion
const daoAppClear = `#pragma version 6
```

File: src/pages/CreateDao.vue [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+		const walletStore = reactive(WalletStore());
+		return {
+			formState,
+			CreateDAOStore,
+			walletStore,
+			validateMessages: VALIDATE_MESSAGES,
+		};
+	},
+	methods: {
+		convertToSeconds(type: DurationType, value: number): number {
```
Comment (rmeena840) [inline src/pages/CreateDao.vue]: Move this method inside 'utility/dateFormatter.utility.ts'

**Review by rmeena840 — APPROVED** [review]
Pre-approving!! Good work. Check the comments before the final merge.

File: src/constants/constant.ts [hunk 3]
```diff
@@ -18,6 +18,9 @@ export const GLOBAL_STATE_MAP_KEY = {
 };
 
 export const DAY_TO_MILLISECONDS = 1000 * 60 * 60 * 24;
+export const DAY_TO_SECONDS = 24 * 60 * 60;
+export const HOUR_TO_SECONDS = 60 * 60;
+export const MINUTE_TO_SECONDS = 60;
```
Comment (vuvoth) [inline src/constants/constant.ts]:
```suggestion
export const DAY_TO_MILLISECONDS = 1000 *DAY_TO_SECONDS;
export const DAY_TO_SECONDS = 24 * HOUR_TO_SECONDS;
export const HOUR_TO_SECONDS = 60 * MINUTE_TO_SECONDS;
export const MINUTE_TO_SECONDS = 60;
```

File: src/contract/dao/dao-app-clear.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    int 1
+    return`;
+
+export const getCompiledDaoClear = async () => {
```
Comment (vuvoth) [inline src/contract/dao/dao-app-clear.ts:7]: Let export this file in dao/index.ts

**Review by vuvoth — APPROVED** [review]
Nice work! I wonder about `contract/dao`... Maybe we will make it more better in future.

File: src/constants/constant.ts (hunk 3 above)
Comment (Megha-Dev-19) [reply src/constants/constant.ts]: Nice suggestion. will update

File: src/contract/dao/dao-app-clear.ts (hunk 4 above)
Comment (Megha-Dev-19) [reply src/contract/dao/dao-app-clear.ts:7]: okay

**Pull request merged**